CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing
```

Comment:
A little extensive in my eyes ... also dont think we have to copy other peoples docs

Reply:
As long as credited, it's fine. Feel free to express yours.